server/scripts/register_github_app.py

```
@@ -0,0 +1,488 @@
+        pattern = re.compile(rf"^{re.escape(key)}=.*$", re.MULTILINE)
+        replacement = f"{key}={value}"
+        if pattern.search(text):
+            text = pattern.sub(replacement, text)
+        else:
+            text = text.rstrip() + f"\n{replacement}\n"
+    ENV_PATH.write_text(text)
+
+
+def store_secret_in_vault(path: str, field: str, value: str) -> bool:
+    try:
+        subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)